src/lib/popover/popover-foundation.ts

```diff
@@ -526,6 +536,16 @@ export class PopoverFoundation extends BaseClass implements IPopoverFoundation {
     }
   }
 
+  public get delay(): number {
+    return this._delay;
+  }
+  public set delay(value: number) {
+    if (this._delay !== value) {
+      this._delay = value;
```

Comment:
Let's add some safe guards for `NaN` and negative numbers here and just use the default hover delay if detected. May be worth updating the tooltip `delay` property to guard against that the same way as well.

```typescript
public set delay(value: number) {
  if (isNaN(value) || value < 0) {
    value = POPOVER_CONSTANTS.defaults.hoverDelay;
  }
  if (this._delay !== value) {
    // ...
  }
}
```

We should add a test that captures trying to set these values as well.